src/mixins/name-request-mixin.ts

```diff
@@ -9,20 +9,19 @@ import { NameRequestIF } from '@/interfaces'
 @Component({})
```

Comment:
This file is now exactly the same as in Create UI... in preparation for moving it to shared components library.